src/node_os.cc

```diff
@@ -261,7 +261,8 @@ static void GetInterfaceAddresses(const FunctionCallbackInfo<Value>& args) {
     // contain non-ASCII characters.  On UNIX, it's just a binary string with
     // no particular encoding but we treat it as a one-byte Latin-1 string.
 #ifdef _WIN32
-    name = String::NewFromUtf8(env->isolate(), raw_name);
+    name = String::NewFromUtf8(env->isolate(), raw_name,
+        v8::NewStringType::kNormal).ToLocalChecked();
 #else
     name = OneByteString(env->isolate(), raw_name);
 #endif
```

Comment:
Side note: While it may be true that UNIX systems are somewhat encoding-agnostic here, it’s more than reasonable to assume UTF8 as the default as well. It’s what people will expect if they name the interface from any input that uses UTF-8, which should be the most frequent case by far these days.

I’d suggest either merging these directly in this PR, since you’re already touching the code, or leaving a TODO comment about it.